src/transformStateWithClones.js

```diff
@@ -24,8 +24,8 @@ function transformStateWithClones(state, actions) {
         break;
 
       case 'removeProperties':
-        for (const d of el.keysToRemove) {
-          delete newObj[d];
+        for (const dele of action.keysToRemove) {
```

Comment:
it is not recommended to shorten the names of your variables. They should have maximally clear meaning. `property` would sounds much better here